libbeat/_meta/fields.ecs.yml

```diff
@@ -1,5 +1,5 @@
 # WARNING! Do not edit this file directly, it was generated by the ECS project,
-# based on ECS version 1.6.0.
+# based on ECS version 1.7.0.
```

Comment:
@ebeahan This is an import of the 1.7 + experimental changes.

I think it may be worth tweaking the header in our artifact to reflect that, WDYT?

Reply:
I did do a slight no-no and modified it to say `1.7.0` v. `1.7.0-dev` -- but yeah, currently the experimental builds have nothing that indicates in the manifests that they're "experimental"

Reply:
Yeah I saw that 🧐 😉

Reply:
I believe it's pulling from the `version` defined in the branch the artifacts are generated?

https://github.com/elastic/ecs/blob/1.7/experimental/generated/beats/fields.ecs.yml#L2

Reply:
Yep: [from the generator](https://github.com/elastic/ecs/blob/6cbb5448edf921ed9b8b797bd8310003f109259a/scripts/generator.py#L89-L97). Wondering if it makes sense to add some of the `experimental` detection stuff from the [schema loader](https://github.com/elastic/ecs/blob/6cbb5448edf921ed9b8b797bd8310003f109259a/scripts/schema/loader.py#L60) into the generator script. That way you can have the context of whether this is an experimental build or not from the get-go -- then you'd be able to generate a string like `ECS version 1.7.0 (experimental)` in all of the generated files or something.

Either way, apart from people upgrading ECS in beats and not knowing we take from the experimental folder, this probably isn't that big of a deal.